processor/awsapplicationsignalsprocessor/factory.go

```diff
@@ -0,0 +1,91 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package awsapplicationsignalsprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/awsapplicationsignalsprocessor"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/processor"
+	"go.opentelemetry.io/collector/processor/processorhelper"
+)
+
+const (
+	// The stability level of the processor.
+	stability = component.StabilityLevelBeta
```

Comment:
This should be `Development` while it is performing a NoOp

Reply:
Changed to `StabilityLevelDevelopment`